docs/client-libraries-cluster-level-failover.md

```diff
@@ -38,7 +38,12 @@ This is an example of how to construct a Java Pulsar client to use automatic clu
 private PulsarClient getAutoFailoverClient() throws PulsarClientException {
     String primaryUrl = "pulsar+ssl://localhost:6651";
     String secondaryUrl = "pulsar+ssl://localhost:6661";
+    String primaryTlsTrustCertsFilePath = "primary/path";
     String secondaryTlsTrustCertsFilePath = "secondary/path";
+    Authentication primaryAuthentication = AuthenticationFactory.create(
```

Comment:
The primaryAuthentication is in pulsarClient initiation and we need to add a configuration for pulsar client creation. Otherwise, users will be confused by `primaryAuthentication` and `primaryTlsTrustCertsFilePath` initiated without use.
```
PulsarClient pulsarClient = PulsarClient.builder()
            .authentication(primaryAuthentication)
            .tlsTrustCertsFilePath(primaryTlsTrustCertsFilePath)
            .serviceUrl("pulsar://localhost:6650")
            .build();
````